src/database/database_reader.py

```diff
@@ -8,19 +8,33 @@ def __init__(self, db_name, coll_name):
         self.db = self.db_manager.get_database(db_name)
         self.collection = self.db_manager.get_collection(db_name, coll_name)
 
-    def execute_query(self, query, columns, use_id=0, limit=0):
+    def execute_query(self, query=None, columns=None, use_id=0, limit=0):
         """ Make a query for specified collection and return result as a list """
+        if columns is None:
+            columns = []
+        if query is None:
+            query = {}
         fields_selected = {}
         for name in columns:
             fields_selected[name] = 1
         fields_selected['_id'] = use_id
         return list(self.collection.find(query, fields_selected).limit(limit)) # maybe changed to another format
 
-    def get_nth_chunk(self, query, columns, use_id=0, chunk_size=0, chunk_number=0):
+    def get_nth_chunk(self, query=None, columns=None, use_id=0, chunk_size=0, chunk_number=0):
         """ Returns a n-th chunk of data from database, chunks are indexed from 0 """
+        if columns is None:
+            columns = []
+        if query is None:
+            query = {}
         fields_selected = {}
         for name in columns:
             fields_selected[name] = 1
         fields_selected['_id'] = use_id
         chunk = self.collection.find(query, fields_selected).skip(chunk_size*chunk_number).limit(chunk_size)
-        return pd.DataFrame(list(chunk))
\ No newline at end of file
+        return pd.DataFrame(list(chunk))
+
```

Comment:
If columns=None and use_id=1 we get only column with _id. I'm not sure that is excepted behave of function.

Reply:
Will be ok if I include id in list of columns on every function?

Reply:
When I was writing it I meant to exclude id by default, but empty list of columns should work exactly the same, I'll check it later.

Reply:
I think this is good to exclude id by default. For me more natural is to get all columns when columns=None and use_id=1, but this is only my feeling.